e2e/e2e.go

```diff
@@ -47,28 +48,17 @@ var (
 
 func init() {
 	// Client/Scheme setup.
-	err := clientgoscheme.AddToScheme(Scheme)
-	if err != nil {
-		panic(err)
-	}
-
-	err = aoapis.AddToScheme(Scheme)
-	if err != nil {
-		panic(err)
-	}
-
-	err = apiextensionsv1.AddToScheme(Scheme)
-	if err != nil {
-		panic(err)
-	}
-
-	err = operatorsv1.AddToScheme(Scheme)
-	if err != nil {
-		panic(err)
-	}
+	addAPIsToScheme(Scheme, []func(s *runtime.Scheme) error{
```

Comment:
Nice!
This is already implemented with `runtime.SchemeBuilder` see https://github.com/openshift/addon-operator/blob/main/apis/addons/addons.go#L10, no need for a custom implementation.

Reply:
Thanks!